### PR TITLE
Gaurav6982

### PR DESCRIPTION
**Pull request by Gaurav6982**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have Added Files for Rectangle Cutting using DP WITH CPP and JAVA.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by Gaurav6982** [comment]
Please Review @plazzy99

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Gaurav6982 Do not commit unnecessary files i.e. `bin_exp.js `.

**Comment by Gaurav6982** [comment]
> @Gaurav6982 Do not edit unnecessary files i.e. `bin_exp.js `.
I have not made a change in that file, actually that was my previous commit which got added itself. I think I have made some error while making a pull request. Sorry for that.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Gaurav6982 Still `bin_exp.js` files is there please do not commit these files.

File: JavaScript/math/bin_exp.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (vatsalkesarwani12) [inline JavaScript/math/bin_exp.js]: remove this file from this pull request.

**Pull request closed without merging**